app/templates/direct-award/end-search.html

```diff
@@ -29,49 +29,61 @@
 {% endblock %}
 
 {% block main_content %}
+{% if errors %}
+  <div class="validation-masthead" aria-labelledby="validation-masthead-heading" aria-role="group" tabindex="-1">
+    <p>
+      You need to <a href="#user_understands">confirm that you’ve finished editing your search</a> before you
```

Comment:
It would be nice to do something more like the toolkit here with `a href="#{{ form.user_understands.id }}"`.